gateway/ln-gateway/src/lib.rs

```diff
@@ -672,110 +669,110 @@ impl Gateway {
         let GatewayState::Running { lightning_context } = self.get_state().await else {
             panic!("Gateway isn't in a running state")
         };
+
         loop {
-            match stream.next().await {
-                Some(Ok(htlc_request)) => {
-                    info!(
-                        "Intercepting HTLC {}",
-                        PrettyInterceptHtlcRequest(&htlc_request)
-                    );
-                    if handle.is_shutting_down() {
-                        break;
-                    }
+            let htlc_request = match stream.next().await {
+                Some(Ok(htlc_request)) => htlc_request,
+                other => {
+                    info!("Got {other:?} while handling HTLC stream, exiting from loop...");
```

Comment:
This can probably be a `warn` or `error`

e.g `warn!(?other, "Unexpected response from HTLC stream, exiting from loop...");`

Reply:
Done.